CMakeLists.txt

```diff
@@ -208,7 +208,7 @@ link_directories(
 
 set(OPENCL_CLANG_LINK_LIBS ${CMAKE_DL_LIBS})
 
-if(NOT LLVMSPIRVLib IN_LIST LLVM_AVAILABLE_LIBS)
+if(NOT LLVMSPIRVLib IN_LIST LLVM_AVAILABLE_LIBS OR (USE_PREBUILT_LLVM AND LLVM_LINK_LLVM_DYLIB))
```

Comment:
Does it mean that if LLVM was built with `-DLLVM_LINK_LLVM_DYLIB=ON`, then `LLVMSPIRVLib` **is** in `LLVM_AVAILABLE_LIBS`?

Reply:
Looks like that is the case. We have pre-built llvm with LLVM_LINK_LLVM_DYLIB=ON and LLVMSPIRVLib is built in-tree and is in LLVM_AVAILABLE_LIBS but not linked to libLLVM.

Reply:
Ok